.gitignore

```
@@ -62,6 +62,9 @@ build/
 [Ll]ogs/
 out/
 
+# Builds may generate a CCache directory.
+ccache
```

Nit: The other directories all end with `/`, may as well keep the pattern going.
```suggestion
ccache/
```